test/data/config.template.yaml

```diff
@@ -0,0 +1,7 @@
+name: node/install-test
+version: 1.2.0
+maintainer: foo@bar.com
+description: Template to install and test npm packages
+steps:
```

Comment:
Based on stjohn's suggestion, we should have this at `config` level. Can you take a look at the updated issue? Also there are other fields in here.